src/components/accordion-item/accordion-item.tsx

```diff
@@ -172,7 +200,13 @@ export class AccordionItem implements ConditionalSlotComponent {
 
   render(): VNode {
     const dir = getElementDir(this.el);
-    const iconEl = <calcite-icon class={CSS.icon} icon={this.icon} scale="s" />;
+    const iconStartEl = this.iconStart ? (
+      <calcite-icon class={CSS.iconStart} icon={this.iconStart} key={CSS.iconStart} scale="s" />
```

Comment:
Interesting using the CSS class for the key :)

I'm ok with it, just wondering if we should allow it or not? 🤷

Reply:
Thought of using `guid` from util classes, but these are unique to the component which could be used a key . I can change it if the pattern is inconsistent.

Reply:
Maybe just set the keys to `"icon-start"` since using the classes could mess things up if a class changes.